servicecontrol/upgrades/2to3.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Upgrade ServiceControl from version 2 to version 3
+summary: Instructions on how to upgrade ServiceControl from version 2 to 3
+isUpgradeGuide: true
+reviewed: 2018-08-17
+---
+
+## Overview
+
+Upgrading ServiceControl from version 2 to version 3 is a major upgrade and requires careful planning. During the upgrade process, the instance of ServiceControl that is being upgraded will no longer be available and will not be ingesting any messages. The upgrade process is not reversible and cannot be reversed once started.
```

Comment:
This upgrade can be reversed. The users can uninstall v3 and drop back to a v2 instance. The irreversible part for the v1->v2 upgrade was because the database changed.